zerver/webhooks/gogs/doc.html

```diff
@@ -0,0 +1,41 @@
+
+<p>
+    First, create the stream you'd like to use for Gogs Webhook notifications,
+    and subscribe all interested parties to this stream. The integration will
+    use the default stream <code>commits</code> if no stream is supplied in
+    the hook; you still need to create the stream even if you are using this
+    default.
+</p>
+
+<p>Next, go to your repository page and click <b>Settings</b>:</p>
+<p><img src="/static/images/integrations/gogs/001.png"/></p>
+
+<p>From there, select <b>Webhooks</b>:</p>
+<p><img src="/static/images/integrations/gogs/002.png"/></p>
+
+<p>Click <b>Add webhook</b> and then <b>Gogs</b>.</p>
+<p><img class="screenshot" src="/static/images/integrations/gogs/003.png"/></p>
+
+<p>Authorize yourself and configure your webhook.</p>
+
+<p>In the <b>Payload URL</b> field, enter a URL constructed like this:</p>
+
+<p><code>{{ external_api_uri_subdomain }}/v1/external/gogs?api_key=abcdefgh&amp;stream=gogs</code></p>
+
+<p>where <code>api_key</code> is the API key of your Zulip bot.</p>
+
+<p>Then, set <b>Content type</b> to <code>application/json</code>.</p>
+
+<p>
+    Next, select the actions that you want to result in a
+    Zulip notification and click <b>Add Webhook</b>.
```

Comment:
"Next, select the events that you want to trigger Zulip notifications. After you have selected all the desired events, click the <b>Add Webhook</b> button."